frontend/src/metabase/visualizations/echarts/cartesian/chart-measurements/index.ts

```diff
@@ -159,15 +225,46 @@ export const getChartPadding = (
   }
 
   const yAxisNameTotalWidth =
-    CHART_STYLE.axisName.size / 2 + CHART_STYLE.axisNameMargin;
+    CHART_STYLE.axisName.size + CHART_STYLE.axisNameMargin;
 
-  if (chartModel.leftAxisModel?.label) {
-    padding.left += yAxisNameTotalWidth;
-  }
-  if (chartModel.rightAxisModel?.label) {
-    padding.right += yAxisNameTotalWidth;
+  const currentBoundaryWidth =
+    chartWidth -
+    padding.left -
+    padding.right -
+    yAxisNameTotalWidth -
```

Comment:
We should consider `yAxisNameTotalWidth` based on the numer of axes and whether they have name displayed, that's why previously we explicitly checked for each axis label existence